wsclient/src/test/java/org/hpccsystems/ws/client/BaseWsAttributesClientIntegrationTest.java

```diff
@@ -91,28 +68,28 @@ public void testWriteAttributes() throws Exception
                     new String(Files.readAllBytes(testdir.listFiles()[i].toPath()))));
         }
 
-        List<ECLAttributeInfo> found=client.findItems(testmodname, null,null,null,null,null);
+        List<ECLAttributeInfo> found=wsattclient.findItems(testmodname, null,null,null,null,null);
         if (found.size()>0)

```

Comment:
how will this test function when run against clusters like http://10.173.147.1:8010 that have no wsattributes service on port 8145? Is there a check somewhere to bypass this test?

Reply:
@drealeed take a look at the latest commit. All tests targeting non eclwatch services will need to query the dedicated port for that service.
If this test shouldn't execute with its group  (IntegrationTest), we might need to create a new (ExtendedIntegrationTest) or something similar to that.